test/generic/Residue-test.jl

```diff
@@ -67,6 +67,8 @@ end
 
    @test isa(d, EuclideanRingResidueRingElem)
 
+   @test R(2//3) * R(3) == R(2)
```

Comment:
I note that this also works in Oscar:
```
julia> F = GF(5);

julia> F(2//3)
4
```
And of course GAP also supports all that, so this change is fine from my side... But let's see if @thofma or @fieker agree / disagree.

Reply:
Fine with me.